src/screens/TopicPageScreen/index.js

```diff
@@ -37,15 +37,15 @@ const styles = StyleSheet.create({
     backgroundColor: 'lightblue',
     justifyContent: 'center',
     alignItems: 'center',
-    paddingTop: 40,
+    paddingTop: dimen.normalizeDimen(40),
     position: 'absolute',
     top: 0,
     left: 0,
     right: 0,
     zIndex: 2
   },
   headerText: {
-    fontSize: 20,
+    fontSize: normalizeFontSizeByWidth(20),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hard-coded padding-top value and font size have been replaced with normalized values using utility functions. This is a good practice as it ensures consistency across different screen sizes and resolutions.

```diff
-    paddingTop: 40,
+    paddingTop: dimen.normalizeDimen(40),
...
-    fontSize: 20,
+    fontSize: normalizeFontSizeByWidth(20),
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->